src/services/RecipeService.js

```diff
@@ -23,6 +23,21 @@ class RecipeService {
     const route = `${routes.RECIPE}/${routes.RECS}`;
     return getHelper(route);
   };
+
+  getLikedRecipes = () => {
+    const route = `${routes.RECIPE}/${routes.LIKE}`;
+    return getHelper(route);
+  }
+
+  getRecipeToLike = (recipeId) => {
+    const route = `${routes.RECIPE}/${routes.LIKE}/${recipeId}`;
+    return getHelper(route);
```

Comment:
I believe the like route is POST, and the unlike route is DELETE. Look at the back end for the signature. They return collections I think

Reply:
good catch